Project/src/main/resources/templates/editor.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <title>Dashboard</title>
-    <script src="webjars/jquery/1.9.1/jquery.min.js"></script>
+    <script src="webjars/jquery/1.9.1/jquery.js"></script>
```

Comment:
This should still be able to be minified. Something else must be wrong if you're getting a 404 while trying to retrieve this. Here's what I could potentially see could be the problem
```suggestion
    <script src="/webjars/jquery/1.9.1/jquery.min.js"></script>
```